### ISSUE TITLE
references to documentation have changed

### ISSUE THREAD

**Issue by alensiljak** [nuex/t#10]
Pretty much all the references to the documentation (ledger, hledger) in ReadMe are now outdated and lead to 404 pages.

**Comment by egwynn** [comment]
Yep. There are two links which are broken. One is repeated a few times and the other only once. Here's how they should be fixed:

| Old (broken) | New (working as of this posting) |
|---|---|
| ~~http://ledger-cli.org/2.6/ledger.html#Using-timeclock-to-record-billable-time~~ | https://www.ledger-cli.org/2.6/ledger.html#Using-timeclock-to-record-billable-time |
| ~~https://hledger.org/MANUAL.html#timelog-files~~ | https://hledger.org/manual.html#timeclock-format |
